Display/features/steps/gets_sensor_data.py

```diff
@@ -0,0 +1,76 @@
+import os
+from behave import given, when, then
+
+import server
+from sensors import Sensors, NotEnoughSensors
+import constants
+
+
+@given("there is an object that represent the total sensing package")
+def step_impl(context):
+    os.environ[constants.SENSOR_QUANTITY] = constants.ENOUGH_SENSORS
+    context.sensors = server.Sensors(1, 1, 1, 1)
+
+
+@when("these sensors are initialized")
+def step_impl(context):
+    pass
```

Comment:
are sensors automatically initialilzed?

Reply:
My thinking was that the drivers would initialize would initialize the sensors and that that step would be hidden from the person using the `sensors` module.  But I'm not totally sure.  This is one of those instances where I got stuck thinking I need to write the test and define the code, but I need an idea of what the code is going to be in order to write the test.